src/components/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import '../style/header.css'
+
+const Header = ({ handleSearch }) => {
+
+  const clearSearch = (e) => e.target.parentElement.firstChild.lastChild.value = ''
+
+  return (
+    <div className='header_container'>
+      <a href='http://localhost:3000/'>Home</a>
```

Comment:
if you publish this in a hosting service, the url won't be localhost

```suggestion
      <a href='/'>Home</a>
```